libparsec/crates/serialization_format/README.md

```diff
@@ -0,0 +1,37 @@
+This create provides function-like [`procedural macros`] for generating
```

Comment:
Could you add a heading title ?

```suggestion
# Libparsec Serialization Format

This create provides function-like [`procedural macros`] for generating
```

Reply:
Adding a heading title result in an extra title in Rust docs:
![image](https://github.com/Scille/parsec-cloud/assets/786907/0cc6ee9a-e379-4693-9984-36c22cd39006)

Instead of just:
![image](https://github.com/Scille/parsec-cloud/assets/786907/c6ba431f-eeba-4dd5-abda-179cd33025ed)

I guess it is acceptable... maybe using a different title?